Monoisome/Monoisome.fea

```diff
@@ -114,569 +114,567 @@ feature liga {
   sub f a hyphen p l u s by plus.fa;
   sub f a hyphen m i n u s by minus.fa;
   sub f a hyphen a s t e r i s k by asterisk.fa;
-  sub f a hyphen e x c l a m a t i o n hyphen c i r c l e by ;
-  sub f a hyphen g i f t by ;
-  sub f a hyphen l e a f by ;
-  sub f a hyphen f i r e by ;
-  sub f a hyphen e y e by ;
-  sub f a hyphen e y e hyphen s l a s h by ;
-  sub f a hyphen w a r n i n g by ;
-  sub f a hyphen e x c l a m a t i o n hyphen t r i a n g l e by ;
-  sub f a hyphen p l a n e by ;
-  sub f a hyphen c a l e n d a r by ;
-  sub f a hyphen r a n d o m by ;
-  sub f a hyphen c o m m e n t by ;
-  sub f a hyphen m a g n e t by ;
-  sub f a hyphen c h e v r o n hyphen u p by ;
-  sub f a hyphen c h e v r o n hyphen d o w n by ;
-  sub f a hyphen r e t w e e t by ;
-  sub f a hyphen s h o p p i n g hyphen c a r t by ;
-  sub f a hyphen f o l d e r by ;
-  sub f a hyphen f o l d e r hyphen o p e n by ;
-  sub f a hyphen a r r o w s hyphen v by ;
-  sub f a hyphen a r r o w s hyphen h by ;
-  sub f a hyphen b a r hyphen c h a r t hyphen o by ;
-  sub f a hyphen b a r hyphen c h a r t by ;
```

Comment:
I may have accidentally deleted it thinking it was duplicated. Do you see others missing as well? I deleted a few that I thought were duplicate rows.
I will check any missing lines you find first thing in the morning (PST time). Sorry if that caused trouble.